enc_speed.py

```diff
@@ -0,0 +1,19 @@
+from nupic.encoders.scalar import ScalarEncoder
+from nupic.encoders.random_distributed_scalar import RandomDistributedScalarEncoder as RDSE
+
+import numpy as np
+
+total=10000
```

Comment:
Need spaces on either side of `=` per [coding style guide](https://github.com/numenta/nupic/wiki/Python-Style-Guide#spaces)